### DISCUSSION TITLE
Configuration de la puissance dans attributs de base.

### DISCUSSION THREAD

**Discussion by swknf** [jmcollin78/versatile_thermostat#574] [Q&A] [answered]
Bonjour, j'ai une question car je ne suis pas certains de bien cerner le fonctionnement de l'attribut puissance dans le cas de la présence de multiple convecteur pour le même vtherm.

J'ai une pièce disposant de 3 convecteurs de 1500w chacun. 
J'ai donc créé un vtherm de type switch pour piloter les modules nodon qui se trouvent derrière chaque convecteur. 
Je définis donc dans les entités sous jacente mes 3 convecteurs. Jusque là pas de problème.
J'ai compris que les convecteurs ne seraient allumés qu'à tour de rôle et pas en simultané, du coup je me pose la question suivante :
dans l'attribut puissance :
dois-je définir la puissance sur le total cumulé des 3 convecteurs soit : 4500W 
ou bien ( vu qu'un seul sera allumé ) à la puissance d'un convecteur individuel soit : 1500W ?

Je n'ai pas rien trouvé d'explicite dans la documentation ( mais je suis peut être simplement passé à côté ).

Merci de votre aide :)

**Comment by gael1980** [comment, marked as answer]
Bonjour,
Il faut mettre la puissance totale cumulée des 3 convecteurs soit : 4500 W
Gael

**Reply by swknf** [reply]
Impec Merci

**Comment by jmcollin78** [comment]
Hello @swknf 

Ci-joint un extrait du readme :
![Capture d’écran 2024-10-26 à 11 56 45](https://github.com/user-attachments/assets/65fae6b0-9dc5-409f-ac6f-85f237592158)

**Reply by swknf** [reply]
Ha bah merci, effectivement je n'avais pas les yeux en face des trous....
